src/plugins/discover/public/application/apps/main/utils/nested_fields.ts

```diff
@@ -51,7 +52,8 @@ export function isNestedFieldParent(fieldName: string, indexPattern: IndexPatter
     !!indexPattern.fields.getAll().find((patternField) => {
       // We only want to match a full path segment
       const nestedRootRegex = new RegExp(escapeRegExp(fieldName) + '(\\.|$)');
-      return nestedRootRegex.test(patternField.subType?.nested?.path ?? '');
+      const subTypeNested = getDataViewFieldSubtypeNested(patternField);
+      return nestedRootRegex.test(subTypeNested?.nested.path ?? '');
```

Comment:
we can be sure, that `.nested` exists in this case, right?

Reply:
`getDataViewFieldSubtypeNested` returns `IFieldSubTypeNested | undefined`